autoendpoint/src/extractors/notification_headers.rs

```diff
@@ -351,32 +340,7 @@ mod tests {
         assert_encryption_error(result, "Missing Content-Encoding header");
     }
 
-    /// Valid 01 draft encryption passes validation
-    #[test]
-    fn valid_01_encryption() {
-        let req = TestRequest::post()
-            .header("TTL", "10")
-            .header("Content-Encoding", "aesgcm128")
-            .header("Encryption", "salt=foo")
-            .header("Encryption-Key", "dh=bar")
-            .to_http_request();
-        let result = NotificationHeaders::from_request(&req, true);
-
-        assert!(result.is_ok());
-        assert_eq!(
-            result.unwrap(),
-            NotificationHeaders {
-                ttl: 10,
-                topic: None,
-                encoding: Some("aesgcm128".to_string()),
-                encryption: Some("salt=foo".to_string()),
-                encryption_key: Some("dh=bar".to_string()),
-                crypto_key: None
-            }
-        );
-    }
-
-    /// Valid 04 draft encryption passes validation
+    /// Valid 04 draft encryption passes validationgit o
```

Comment:
nit: some cruft ended up at the end of this comment

Reply:
crap. Thanks!